apps/datadogcharts/datadog_charts.star

```diff
@@ -0,0 +1,160 @@
+    query = first_widget.get("definition").get("requests")[0].get("queries")[0].get("query")
+
+    # Query metrics API to get the list of points to plot
+    from_time = time.now().unix - 3600
+    to_time = time.now().unix
+    print("Making query to DataDog API: ", query, str(from_time), str(to_time))
+    query_response = http.get(
+        "https://api.datadoghq.com/api/v1/query",
+        params = {"from": str(from_time), "to": str(to_time), "query": query},
+        headers = {"DD-API-KEY": DD_API_KEY, "DD-APPLICATION-KEY": DD_APP_KEY, "Accept": "application/json"},
+    ).json()
+
+    # Check if we got data, if not, show an error
+    if len(query_response.get("series")) == 0:
+        print("No data returned from query.")
+        print(query_response)
+        child = render.Row(
+            cross_align = "center",
+            main_align = "center",
+            children = [
+                render.WrappedText(content = "No data returned from query."),
+            ],
+        )
+        return render.Root(child = child)
+
+    # Get the data from the response
+    raw_points = query_response.get("series")[0].get("pointlist")
+
+    # Convert datapoints into List[Tuple[float, float]]
+    datapoints = []
+    for point in raw_points:
+        datapoints.append((point[0], point[1]))
+
+    return redener(title, datapoints)
+
+def redener(display_name, datapoints):
+    logo = render.Image(src = DATADOG_ICON, width = 16, height = 16)
+    text = render.Marquee(
+        width = 48,
+        child = render.Text(content = display_name, font = "6x13", color = "#fff"),
+    )
+    plot_row = render.Row(
+        expanded = True,
+        main_align = "space_evenly",
+        cross_align = "end",
+        children = [
+            render.Plot(
+                data = datapoints,
+                fill = True,
+                width = 60,
+                height = 24,
+                color = "#0f0",
+                color_inverted = "#f00",
+            ),
+        ],
+    )
+
+    empty_row = render.Column(
+        main_align = "space_between",
+        cross_align = "center",
+        children = [],
+    )
+
+    top_row_columns_left = render.Column(
+        expanded = True,
+        main_align = "space_between",
+        cross_align = "center",
+        children = [logo],
+    )
+    top_row_columns_right = render.Column(
+        expanded = True,
+        cross_align = "center",
+        children = [text],
+    )
+
+    top_row = render.Row(
+        expanded = True,
+        main_align = "space_between",
+        cross_align = "center",
+        children = [top_row_columns_left, top_row_columns_right],
+    )
+    bottom_row = render.Column(
+        expanded = True,
+        main_align = "space_between",
+        cross_align = "center",
+        children = [empty_row, plot_row],
+    )
+
+    root = render.Stack(
+        children = [bottom_row, top_row],
+    )
+
+    return render.Root(child = root)
```

Comment:
```suggestion
    return render.Root(child = root)

def get_schema():
    return schema.Schema(
        version = "1",
        fields = [
            schema.Text(
                id = "api_key",
                name = "API Key",
                desc = "DataDog API Key",
                icon = "key",
            ),
            schema.Text(
                id = "app_key",
                name = "App Key",
                desc = "DataDog App Key",
                icon = "key",
            ),
            schema.Text(
                id = "dashboard_id",
                name = "Dashboard ID",
                desc = "DataDog Dashboard ID",
                icon = "key",
            ),
        ],
    )

```

Reply:
This adds a schema to the app, so that users can configured it in the Tidbyt iOS/Android app. You can also test it out locally via `pixlet serve apps/datadogcharts/datadog_chart.star`.